homework/src/index.js

```diff
@@ -30,18 +35,97 @@
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { html: err.message, class: 'alert-error' });
-      } else {
-        createAndAppend('pre', root, { html: JSON.stringify(data, null, 2) });
-      }
-    });
+  const HYF_REPOS_URL = 'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
+  const root = document.getElementById('root');
```

Comment:
You should place this statement definition `root` inside function `main()` so that it is guaranteed to be executed through `window.onload`. The `root` div may not have been created by the browser yet when this statement is executed as this point. The 'load' event is guaranteed to executed when the web page and all its resources are completely loaded.

Reply:
It is inside the main function now